client/src/containers/analysis-visualization/analysis-filters/years-range/component.tsx

```diff
@@ -23,37 +27,54 @@ const YearsRange: React.FC = () => {
 
   const { startYear, endYear, yearsGap, setYearsRange } = useYearsRange({
     years,
+    yearsGap: 1,
     // Map mode only makes use of the endYear and will display the Select,
     // not the YearsRangeFilter.
-    validateRange: visualizationMode !== 'map',
+    validateRange: visualizationMode !== 'map' && visualizationMode !== 'table',
```

Comment:
Isn't it easier to just validate when `visualizationMode === 'chart'` ?

Reply:
Anyway, why are we excluding the `table` mode of the validation now? This was not the previous behaviour, wasn't it?

Reply:
I was testing a solution and add it, then forgot to remove.